[0.71] Update page on Integration with Existing Apps for 71 (#3446)
facebook/react-native-website: docs/_integration-with-exisiting-apps-java.md

### Creating a release build in Android Studio

You can use Android Studio to create your release builds too! It’s as quick as creating release builds of your previously-existing native Android app.

> If you use the React Native Gradle Plugin as described above, everything will just work from Android Studio.

Reviewer: ```suggestion
If you use the React Native Gradle Plugin as described above, everything should work when running app from Android Studio.
```

Phrasing nit, to address alex check complain.
